common/djangoapps/entitlements/api/v1/tests/test_views.py

```diff
@@ -56,6 +57,18 @@ def _get_data_set(self, user, course_uuid):
             "order_number": "EDX-1001",
         }
 
+    def _assert_default_policy(self, policy):
+        """
+        Assert that a policy is equal to the default Course Entitlement Policy.
+        """
+        default_policy = CourseEntitlementPolicy()
+        assert (
```

Comment:
It might make sense to split these assertions into separate lines so that the test failure output will be more explicit on what part is failing.